docs/api/conventions.md

```diff
@@ -1310,3 +1310,15 @@ export default function Page() {
 [link tag]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link
 [minimatch]: https://www.npmjs.com/package/minimatch
 [handledatarequest]: #entryservertsx
+
+## Environment Variables
+
+Hopefully, your application has at least 3 deployment locations (local, test, and prod). In order to decouple your application from it's eventual deployment location, environment-specific application configuration values are best stored in Environment Variables. How you configure these will vary based on your deployment target in higher environments, so Remix does not do anything special during production runs in your deployment target or via `remix serve`.
```

Comment:
Any use in linking out to something here about env var "best practices"?   Potentially https://12factor.net/config or something similar.

Reply:
🤦‍♂️ Just found the `envvars.md` guide - moving this over there